src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -392,6 +396,49 @@ export function useServicesValidation(): StepValidation {
   };
 }
 
+function validateUserName(userName: string): string {
+  if (!isUserNameValid(userName)) {
+    return 'Invalid user name';
+  }
+  return '';
+}
+
+function validatePassword(
```

Comment:
Can you please use arrow function expressions to be consistent with the rest of the code base?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
This function doesn't validate the password, it returns the error message for an invalid password. Naming is very hard, but a good starting point is to name functions after what they do.